README.Rmd

```diff
@@ -113,99 +113,146 @@ retro_nat_germany <- nat_germany_hosp |>
+Before modelling, the input data needs to be converted into the "reporting triangle" format (see [our model description for details](https://package.epinowcast.org/articles/model)). We also need to determine metadata to facilitate the model specification. This includes the number of days of data to use for the reference and report modules, the maximum delay to consider, and, optionally, a grouping (i.e. age group, location, or both) of observations.
+We process reported data into the format required for `epinowcast` and return it in a `data.table`. At this stage, we need to specify a grouping (i.e age, location) if any. 
 
 ```{r}
 pobs <- enw_preprocess_data(retro_nat_germany, max_delay = 40)
 pobs
 ```
 
-Construct a parametric lognormal intercept only model for the date of reference using the metadata produced by `enw_preprocess_data()`. Note that `epinowcast` uses a sparse design matrix for parametric delay distributions to reduce runtimes so the design matrix shows only unique rows with `index` containing the mapping to the full design matrix.
+The returned output is in the form of a `data.table` with metadata stored as variables. It can be useful to check this output before specifying the model, just to make sure everything is as expected.
+
+### Model specification
+
+The `epinowcast` package is designed to provide users with a flexible and customizable modelling framework. The package comes equipped with several modules that users can utilize to construct models, and also allows users to create their own modules. These ensures that models can be tailored to the user's specific data and context.
+
+#### Default nowcasting model
+
+The default nowcasting model in `epinowcast` consists of three modules: 
+
+- A process (expectation) module that models the expected counts by date of reference (`reference_date`)
+- A parametric reference reporting model which models the reporting delay distribution from the date of reference
+- A non-parametric reporting model which models differences in the reporting delay distribution by date of report (`report_date`), for example, day-of-the-week effects on the reporting delay.
+
+In the following sections, we specify simple models for each of these modules. The appropriateness of these specifications will vary depending on your context. See our vignettes for further details on model specification and examples of more complex models.
+
+#### Process model
+
+A commonly used process model in nowcasting is to model the expected counts by date of reference via a lognormal random walk. This aims to strike a balance between capturing the underlying data generation process while being primarily dependent on the information provided by available observations and their reporting delays. Users may specify this model using the enw_expectation() function. Here, `day` refers to the number of days from the start of the data.
```

Comment:
I see what you are trying to say here (that this is a minimally informed smoothing prior and we thus give lots of influence to the data), but I find the current formulation a bit difficult to understand